charts/osm/templates/cleanup-hook.yaml

```diff
@@ -9,23 +118,26 @@ metadata:
     helm.sh/hook: post-delete
     helm.sh/hook-delete-policy: before-hook-creation,hook-succeeded
 spec:
-  template:
-    metadata:
-      name: {{ .Release.Name }}-cleanup
-      labels:
-        {{- include "osm.labels" . | nindent 8 }}
-    spec:
-      serviceAccountName: {{ .Release.Name }}-cleanup
-      restartPolicy: Never
-      nodeSelector:
-        kubernetes.io/arch: amd64
-        kubernetes.io/os: linux
-      containers:
-        - name: garbage-collector
-          image: bitnami/kubectl
-          imagePullPolicy: IfNotPresent
-          command:
-            - sh
-            - -c
-            - >
-             kubectl delete --ignore-not-found meshconfig -n '{{ include "osm.namespace" . }}' osm-mesh-config;
+  serviceAccountName: {{ .Release.Name }}-cleanup
+  restartPolicy: Never
+  nodeSelector:
+    kubernetes.io/arch: amd64
+    kubernetes.io/os: linux
+  containers:
+    - name: garbage-collector
+      image: bitnami/kubectl
+      imagePullPolicy: IfNotPresent
+      command:
+        - sh
+        - -c
+        - >
+         kubectl delete --ignore-not-found meshconfig -n '{{ include "osm.namespace" . }}' osm-mesh-config;
+         kubectl delete --ignore-not-found meshconfig -n '{{ include "osm.namespace" . }}' osm-mesh-config;
```

Comment:
Why we delete it twice? Do you mean preset-mesh-config?

Reply:
it was accidental I removed it.